build.gradle

```diff
@@ -1,3 +1,7 @@
+wrapper {
+    gradleVersion = '6.8.3'
+    distributionType = Wrapper.DistributionType.BIN
+}
```

Comment:
This code allows for specifying the gradle wrapper version in the build gradle and updating all the generated code by running `./gradlew wrapper` twice

Reply:
All other files in this PR are automatically generated/updated using the gradle wrapper task